test/tests/mortar/displaced-gap-conductance-2d-auxkernel/tests

```diff
@@ -0,0 +1,13 @@
+[Tests]
+  design = 'Constraints/index.md'
+  issues = '#13080'
+  [gap_conductance]
+    type = Exodiff
+    input = 'gap-conductance.i'
+    exodiff = 'gap-conductance_out.e'
+    custom_cmp = 'auxkernel.cmp'
+    mesh_mode = 'REPLICATED'
+    max_parallel = 2
```

Comment:
Any particular reason for the `max_parallel`?

Reply:
No. I'll remove it. I think I only added it because it is a very small test.

Reply:
I believe we should also test a boundary material with Mortar constraints. I think that, as long as the material properties aren't stateful, it should work. I'll add it.